internal/ingestion/component/chunker/tool-py/capture_golden.py

```
@@ -0,0 +1,261 @@
+    return cases
+
+
+def write_manifest() -> None:
+    """Record provenance in one shared file.
+
+    Kept out of the per-case golden files so a re-capture does not touch every
+    fixture's diff with a new timestamp.
+    """
+    try:
+        commit = subprocess.run(
+            ["git", "rev-parse", "HEAD"],
```

Ruff (0.16.0)

[error] Starting a process with a partial executable path

(S607)